SIT.Manager.Avalonia/Views/ServerPage.axaml

```diff
@@ -53,28 +53,41 @@
 				</ScrollViewer>
 			</Grid>
 
-			<StackPanel Grid.Row="1" Margin="10 0 10 10" Orientation="Horizontal">
+			<Grid Grid.Row="1" Margin="10 0 10 10" ColumnDefinitions="*,*,*">
 				<Button Name="StartServerButton" 
-						HorizontalAlignment="Left" 
+						HorizontalAlignment="Stretch" 
 						Margin="0 0 10 0" 
+						Grid.Column="0"
 						ToolTip.Tip="{DynamicResource ServerButtonStartToolTip}"
 						Command="{Binding StartServerCommand}">
-					<StackPanel Orientation="Horizontal">
-						<ui:SymbolIcon Name="StartServerButtonSymbolIcon" Symbol="{Binding StartServerButtonSymbolIcon}" Margin="0 0 10 0"/>
-						<TextBlock Name="StartServerButtonTextBlock" Text="{Binding StartServerButtonTextBlock}"/>
-					</StackPanel>
+					<Grid ColumnDefinitions="auto,*">
+						<ui:SymbolIcon Name="StartServerButtonSymbolIcon" Symbol="{Binding StartServerButtonSymbolIcon}" Margin="0 0 10 0" Grid.Column="0"/>
+						<TextBlock Name="StartServerButtonTextBlock" TextTrimming="CharacterEllipsis" Text="{Binding StartServerButtonTextBlock}" Grid.Column="1"/>
+					</Grid>
 				</Button>			
 				<Button Name="EditServerConfigButton" 
-						HorizontalAlignment="Left" 
-						Margin="0 0 5 0" 
+						HorizontalAlignment="Stretch" 
+						Margin="0 0 10 0" 
+						Grid.Column="1"
 						ToolTip.Tip="{DynamicResource ServerEditServerConfigButtonToolTip}"
 						Command="{Binding EditServerConfigCommand}">
-					<StackPanel Orientation="Horizontal">
-						<ui:SymbolIcon Symbol="Edit" Margin="0 0 10 0"/>
-						<TextBlock Text="{DynamicResource ServerEditServerConfig}"/>
-					</StackPanel>
+					<Grid ColumnDefinitions="auto,*">
+						<ui:SymbolIcon Symbol="Edit" Margin="0 0 10 0" Grid.Column="0"/>
+						<TextBlock Text="{DynamicResource ServerEditServerConfig}" TextTrimming="CharacterEllipsis" Grid.Column="1"/>
+					</Grid>
+				</Button>
+				<Button Name="ClearServerLogsButton"
+						HorizontalAlignment="Stretch"
+						Margin="0 0 10 0"
+						Grid.Column="2"
+						ToolTip.Tip="{DynamicResource ServerClearServerOutputTooltip}"
+						Command="{Binding ClearServerOutputCommand}">
+					<Grid ColumnDefinitions="auto,*">
+						<ui:SymbolIcon Symbol="Delete" Margin="0 0 10 0" Grid.Column="0"/>
+						<TextBlock Text="{DynamicResource ServerClearServerOutputText}" TextTrimming="CharacterEllipsis" Grid.Column="1"/>
+					</Grid>
```

Comment:
Why grid auto * over horizontal stack panel?

Reply:
Because text that was showing to the user was breaking the visual, it's better to use that grid to first fix the visual and if space is not enough for visual it will show textblocks with trimming "..."